app/kumactl/data/install/k8s/metrics/grafana/kuma-dataplane.json

```diff
@@ -454,7 +454,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "envoy_cluster_upstream_cx_total{envoy_cluster_name=\"ads_cluster\",dataplane=\"$dataplane\",mesh=\"$mesh\"}",
+          "expr": "envoy_control_plane_connected_state{dataplane=\"$dataplane\",mesh=\"$mesh\"}",
```

Comment:
According to docs `envoy_control_plane_connected_state` is a boolean, does it mean that we need some aggregating operation to have a number in that target?

Reply:
nope, it works fine this way. I tested it. It's `1.0` and `0.0`. There is no concept of boolean in prometheus metrics.

Reply:
Oh okay, `cx_total` apparently had a value `1` or `0` because proxy always connected to a single control-plane?

Reply:
correct

Reply:
Is it okay to call it `Connections` in that case?

Reply:
Yeah, it's a single connection. Should I use `Connection` and `Active connection to the Control Plane` or do you have a better name for it?

Reply:
Can we name it "Connected", I think it will be clear that it is about control plane?

Reply:
Changed legend from `Connection` to `Connected` and title of pane from `Active connection to the Control Plane` to `Connection to the Control Plane`